README.md

```diff
@@ -239,9 +239,9 @@ Choose wisely the option that fits your needs. Not cancelling previous operation
 
 # Feedbacks and scheduling
 
-Reactive programming is often associated with asynchronous execution. Even though every reactive framework comes with its own GCD abstraction, it is always about saying on which scheduler should the side effect be executed. 
+Reactive programming is often associated with asynchronous execution. Even though every reactive framework comes with its own GCD abstraction, it is always about stating which scheduler the side effect should be executed. 
```

Comment:
Perhaps: “....Should be executed **on**” ?

Reply:
Yes.  I'll update